sh/tests/solutions/grades.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+num_students=$1
+declare -a students
+
+if [ $# -ne 1 ]; then
+    echo "Error: Expect 1 argument only!"
```

Comment:
Here the message is slightly different to the one expected as stated in the subject:
```
Error: expect 1 argument only!
```
In the subject `expect` is lowercase

Reply:
done